test/ball_test.rb

```diff
@@ -2,4 +2,13 @@
 require './lib/ball'
 
 class BallTest < Minitest::Test
-end 
+  def test_it_exists
+    ball = Ball.new('r')
+    assert_instance_of Ball, ball
+  end
+
+  def test_has_attributes
+    ball = Ball.new('r')
+    assert_equal 'r', ball.color
+  end
+end
```

Comment:
Would you want to make a 

```
def setup
end
```

Reply:
you could but ball is pretty simple so setup is overkill.